src/handlers/uniswap/customEvents.ts

```diff
@@ -0,0 +1,132 @@
+            //                 title: 'Liquidity Increased for your owned position', // Short title of the event
+            //                 message: `The liquidity was minted by ${eventData.sender} at the amount ${this.formatUSDCValue(BigNumber.from(eventData.amount))}`,
+            //                 options: {}
+            //             },
+            //         ]
+            //     }
+            // },
+            {
+                eventName: 'SwapEventData',
+                eventHandler: (parsedEventData: ParsedEventData): NotificationDetail[] => {
+                    const eventData = parsedEventData.eventData as SwapEventData;
+
+                    if (eventData.sender === eventData.recipient){
+                        return [
+                            {
+                                address: eventData.recipient,
+                                title: 'Your swap is complete',
+                                message: `The swap you initiated has been completed. Open wallet to see more details.`,
+                                options: {}
+                            }
+                        ]
+                    }
+
+                    return [
+                        {
+                            address: eventData.sender,
+                            title: 'Your swap is complete',
+                            message: `The swap you initiated has been completed`,
+                            options: {}
+                        },
+                        {
+                            address: eventData.recipient,
+                            title: 'Received Swapped Tokens',
+                            message: `You have received tokens from a swap initiated by ${eventData.sender}. Open wallet for more details.`,
+                            options: {}
+                        }
+                    ]
+                }
+            },
+            {
+                eventName: 'BurnEvent',
+                eventHandler: (parsedEventData: ParsedEventData): NotificationDetail[] => {
+                    const eventData = parsedEventData.eventData as BurnEventData;
+                    return [
+                        {
+                            address: eventData.owner,
+                            title: 'Removed from liquidity position',
+                            message: `Your liquidity position has been removed and a total of ${eventData.amount / parseFloat("1000000000000000")} has been returned.`,
+                            options: {}
+                        }
+                    ]
+                }
+            },
+            {
+                eventName: 'MintEvent',
+                eventHandler: (parsedEventData: ParsedEventData): NotificationDetail[] => {
+                    const eventData = parsedEventData.eventData as MintEventData;
+                    if (eventData.owner === eventData.sender){
```

Comment:
The `sender` is the address that minted the liquidity
The `owner` is the owner of the position and recipient of any minted liquidity.

It can be the same wallet or different, so the alert does provide the distinction. Hence the sentence context change when they're the same or different,